pkg/agent/agent.go

```diff
@@ -48,6 +48,8 @@ var agentTypesMap = map[config.AgentType]string{
 	config.TraceabilityAgent: "traceabilityagents",
 }
 
+var CentralClient apic.Client
```

Comment:
Exposing the central client to be used by other agents.

Reply:
There is already a method GetCentralClient() on agent to get the API client, so this is not needed